src/Validation.java

```diff
@@ -0,0 +1,36 @@
+import exception.WrongLoginException;
+import exception.WrongPasswordException;
+
+import java.util.Objects;
+
+public class Validation {
+
+    private String login;
+    private String password;
+    private String confirmPassword;
+
+    public Validation(String login, String password, String confirmPassword) {
+        this.login = login;
+        this.password = password;
+        this.confirmPassword = confirmPassword;
+    }
+
+    public static void checkValidation(String login, String password, String confirmPassword) throws WrongLoginException, WrongPasswordException {
+
+        boolean testLogin = login.matches("^\\w++$");
+        boolean testPassword = password.matches("^\\w++$");
```

Comment:
строку с регулярным выражением лучше вынести в отдельную переменную, а не дублировать

Reply:
также сама регулярка не совсем корректная, зачем в конце 2 +? один должен быть

Reply:
или можно сразу проверять по длине там же
```suggestion
        boolean testLogin = login.matches("^\\w{1,20}$");
        boolean testPassword = password.matches("^\\w{1,20}$");
```